From: Julian Ospald <hasufell@posteo.de>
Date: Tue Sep  8 10:10:18 UTC 2015
Subject: Fix data path fallback
Upstream: no, correct fix pending since 'sys.prefix' is broken in exherbo

--- blink-1.4.1/blink/resources.py
+++ blink-1.4.1/blink/resources.py
@@ -71,7 +71,7 @@
             if os.path.exists(os.path.join(application_directory, 'resources', 'blink.ui')):
                 cls._cached_directory = os.path.join(application_directory, 'resources').decode(sys.getfilesystemencoding())
             else:
-                cls._cached_directory = os.path.join(application_directory, 'share', 'blink').decode(sys.getfilesystemencoding())
+                cls._cached_directory = os.path.join('/usr', 'share', 'blink').decode(sys.getfilesystemencoding())
         return DirectoryContextManager(cls._cached_directory)
 
     @classmethod
